### PR TITLE
[netapp-exporter][manila] ManilaAvailabilityZoneUsageHigh alert update

### PR DESCRIPTION
**Pull request by Carthaca**
back to default, tight capacity situation has improved

### CODE REVIEW

**Comment by Carthaca** [comment]
needs openstack-exporter with https://github.com/sapcc/openstack-exporter/pull/19

File: prometheus-exporters/openstack-exporter/alerts/manila.alerts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+groups:
+- name: manila
+  rules:
+    # 1. substract reserved capacity from reported overall free capacity to get usable capacity
+    # 2. only look at hardware state "live"
+    # 3. sum by host, i.e. look at NetApp cluster, summing up the aggregates aka pools
+    # 4. look at availability zones, joined in trhough manila service metrics
```
Comment (chuan137) [inline prometheus-exporters/openstack-exporter/alerts/manila.alerts:7]: typo "trhough"

File: prometheus-exporters/openstack-exporter/alerts/manila.alerts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  - alert: ManilaAvailabilityZoneUsageHigh
+    expr: count by(zone) (sum by (zone, host)((manila_free_capacity_gb{hardware_state="live"} - (manila_reserved_percentage/100 * manila_total_capacity_gb)) * on(host) group_left(zone) manila_service_state{service="manila-share"}) > 5*1024) < 3 # less than 3 clusters per zone with more than 5 TB free capacity
```
Comment (chuan137) [inline prometheus-exporters/openstack-exporter/alerts/manila.alerts:9]: metrics `(manila_free_capacity_gb{hardware_state="live"} - (manila_reserved_percentage/100 * manila_total_capacity_gb)) * on(host) group_left(zone) manila_service_state{service="manila-share"}` does not seem to work. the labels `host` and `zone` are not matched

**Review by chuan137 — DISMISSED** [review]
please see my inline comments

File: prometheus-exporters/openstack-exporter/alerts/manila.alerts (hunk 1 above)
Comment (Carthaca) [reply prometheus-exporters/openstack-exporter/alerts/manila.alerts]: fixed

File: prometheus-exporters/openstack-exporter/alerts/manila.alerts (hunk 2 above)
Comment (Carthaca) [reply prometheus-exporters/openstack-exporter/alerts/manila.alerts:9]: moved conversation to slack to illustrate on some concrete examples

Comment (chuan137) [reply prometheus-exporters/openstack-exporter/alerts/manila.alerts:9]: metrics deployed only to bronze, and it does work in `na-us-3`

**Comment by Carthaca** [comment]
I applied this to qa, if you want to have a look at this in action

**Review by chuan137 — APPROVED** [review]
LGTM

**Review by joker-at-work — APPROVED** [review]

**Pull request merged**